homeassistant/components/media_player/sonos.py

```diff
@@ -256,3 +263,29 @@ def turn_on(self):
     def play_media(self, media_type, media_id):
         """Send the play_media command to the media player."""
         self._player.play_uri(media_id)
+
+
+class SonosDeviceGroup(SonosDevice):
+    """Wrapper around a group of players.
+
+    Hides separate devices and only cares about whole groups.
+    """
+
+    # pylint: disable=non-parent-init-called, super-init-not-called
+    def __init__(self, hass, group):
+        """Initialize sonos device group."""
+        self.hass = hass
+        MediaPlayerDevice.__init__(self)
```

Comment:
Why don't you just use `super()` here? It should do the right thing and is a bit more obvious to me.